src/foam/nanos/auth/Group.js

```diff
@@ -131,6 +131,19 @@ foam.CLASS({
       documentation: 'Custom authentication settings for this group.'
     }
     */
+    {
+      class: 'Boolean',
+      name: 'flag',
+      documentation: 'flag to filter sme specific groups'
+    },
+    {
+      class: 'String',
+      name: 'label',
```

Comment:
We should be using toSummary to display object data.

Reply:
Just used a label property not toSummary method to set a label value when it is needed. For example, currently 'smeBusinessAdmin', 'smeBusinessApprover', 'smeBusinessEmployee' are used as id but we would like to display as Admin, Approver, Employee. For non sme specific groups, just use its id as default.